.travis/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+TAG="$DEPLOY_SERVER/$DEPLOY_NAMESPACE/$DEPLOY_IMAGE:latest"
+
+echo $DEPLOY_TOKEN | docker login -u $DEPLOY_USER --password-stdin $DEPLOY_SERVER
+docker build -t $TAG .
```

Comment:
Should we run the tests in this container?

Reply:
Is that a thing? Wouldn't we need PG on the container or a remote PG service even if it was?

Reply:
no? I don't understand what you mean

The way Travis deploy section works is it
- only runs on the master branch (so on merge to master)
- only runs if the tests have passed

Reply:
https://docs.travis-ci.com/user/build-stages#specifying-stage-order-and-conditions

I read about someone that was doing exactly this a while back.  They had multiple stages, so they would:
- Build the container image
- Run the tests in a container from the image that was just built
- Push the image if tests passed.

It would give us the advantage that we're running tests on the same platform that we're deploying.  We could also spin up our own postgres container for test that is the same as what we run in production (we've seen differences between Travis PG and production in the past).

Reply:
Also, maybe we could tie in a QA stage and run their test suite against the same container.